### ISSUE TITLE
Unable to create the "Chat with Wikipedia" flow

### ISSUE THREAD

**Issue by ckittel** [Azure-Samples/openai-end-to-end-baseline#15]
This came from a customer reported issue by way of @shoresh5. I'm replicating it here on their behalf.

Reported issue can from following these steps from the deployment guide:

1. Clone an existing prompt flow
1. Click on 'Prompt flow' in the left navigation in Machine Learning Studio
1. Click on the 'Flows' tab and click 'Create'
1. Click 'Clone' under 'Chat with Wikipedia' 
 
The result was:

```output 
Authentication Failed
Unable to access the dag.yaml file with original storage location at Users/<user.name>/promptflow/Flow-created-on-09-20-2024-17-32-35/flow.dag.yaml. Please check the storage setting.
```

**Comment by ckittel** [comment]
For anyone that is running into this, make sure your user has the `Storage File Data Privileged Contributor` (`69566ab7-960f-475b-8e7c-b3118f30c6bd`) role assigned to the ml storage account.  When using the portal, the user's identity is used to access storage.

**Comment by ckittel** [comment]
Fixed: #25

**Comment by AlahmadiQ8** [comment]
For anyone run into this, key based authentication is disabled by default on the storage account. You get this error on AI Foundry because the portal is trying to access storage via authentication  key. 

You must manually enable Key Based Authentication on the storage account. 

<img width="404" alt="Image" src="https://github.com/user-attachments/assets/b23155b8-41c0-44dd-8ef2-d8290ed9c7f2" />
